library/src/main/java/com/devbrackets/android/exomedia/core/listener/OnStateChangeListener.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2016 Brian Wernick,
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.devbrackets.android.exomedia.core.listener;
+
+import com.devbrackets.android.exomedia.core.PlaybackState;
+
+public interface OnStateChangeListener {
+    void onStateChanged(@PlaybackState int playbackState);
```

Comment:
While the `@PlaybackState` is great, the backing integers are owned by ExoPlayer, I'd like internal values (to ExoMedia) that users can reference instead of the ExoPlayer ones.

e.g.
```java
public interface PlaybackState {
    public static final int IDLE = ExoPlayer.STATE_IDLE;
}
```
which would allow implementing apps to check `if (state == PlaybackState.IDLE)...` instead of `if (state == ExoPlayer.STATE_IDLE)...`